src/Http/Connector.php

```diff
@@ -18,8 +19,9 @@
 use Saloon\Traits\Responses\HasCustomResponses;
 use Saloon\Contracts\Connector as ConnectorContract;
 use Saloon\Traits\RequestProperties\HasRequestProperties;
+use Saloon\Contracts\HasDebugging as HasDebuggingContract;
 
-abstract class Connector implements ConnectorContract
+abstract class Connector implements ConnectorContract, HasDebuggingContract
```

Comment:
I'm a bit unsure about this.
We're using a trait that implements the functionality, but it might be considered a breaking change.
I think right now that it's better to document it, and have people add it to their own connectors?
Connectors always has to be created, regardless, so it's less of a task - albeit annoying for something like this.

So maybe remove both trait and interface, in that case?
I'll let you choose, as I'm really unsure about if this is a breaking change or not.

Reply:
I think this is fine for now as I'm using the contract to detect if we need to use it. Hopefully we can fix the middleware ordering in v3 so we can have your original idea of it all being built into a Saloon plugin.